src/strings/ops.c

```diff
@@ -1,6 +1,7 @@
 #include "platform/memmem.h"
 #include "moar.h"
 #define MVM_DEBUG_STRANDS 0
+#define MAX_GRAPHEMES     0xFFFFFFFF
```

Comment:
@samcv I'd rather keep it as MAX_GRAPHEMES. It's not being used for indexing in any way, but it is being used to ensure the resulting string has an allowed/valid value for `MVMStringBody.num_graphs`.